src/docfx/lib/schema/JsonSchema.cs

```diff
@@ -149,5 +149,11 @@ internal class JsonSchema
         /// Properties that are used to indicate the deprecated field
         /// </summary>
         public string ReplacedBy { get; set; }
+
+        /// <summary>
+        /// Properties that are used to indicate the value relationship between two fields
+        /// Mapping relationship: dependentEnums --> <field-name> --> <dependent-field-name> --> <dependent-field-value> --> <allowed-field-values>
+        /// </summary>
+        public Dictionary<string, Dictionary<string, Dictionary<string, string[]>>> DependentEnums { get; set; } = new Dictionary<string, Dictionary<string, Dictionary<string, string[]>>>();
```

Comment:
Json schema uses `enum` and `dependencies`, maybe `enumDependencies` would be a better name for this?

Reply:
Good name